tests/functional/conftest.py

```diff
@@ -359,18 +349,13 @@ def _merge_request(*, source_branch: str):
             time.sleep(0.5)
         assert mr.merge_status != "checking"
 
-        to_delete.append((mr.iid, source_branch))
+        to_delete.extend([mr, mr_branch])
```

Comment:
FYI: `to_delete += mr, mr_branch` doesn't work or at least flake8 complains because `to_delete` is defined outside the scope of this nested function.